src/PagesBody/DogDataPage/DogDataPage.js

```diff
@@ -1,8 +1,16 @@
 import DogData from '../../Organisms/DogData/DogData';
-const randomDogID = 'e8085155-6b82-41f4-a5e5-b4745c332ce2';
+// import { useLocation } from 'react-router-dom';
 
 const DogDataPage = () => {
-  return <DogData id={randomDogID}></DogData>;
+  // From the previous page 'dog-submit' after choosing dog we passed to this dog-data page
+  // dogid by state passed by navigate
+  //      navigate(`./dog-data`, {
+  //         state: { dogId: '' },
+  //       });
+  // const locationPath = useLocation();
+  // const dogId = locationPath.state.dogId;
+  const dogId = 'VIII-40407';
```

Comment:
This shuold be dynamicaly set and consider 2 options:
1. get it from url (somebody getting here through a link) 
2. get it from location.state (like in the example that is commented out)
Both things shuold be resolved in useEffect, for example like this:
`
useEffect(() => {
  if (location.state) {
    setDogId(location.state.dogId)
  } else {
    const currentPath = window.location.pathname;
    const dogId = currentPath.split('/').pop();
    setDogId(dogId)
  }
}, []);
`